docs/installation.md

```diff
@@ -35,49 +41,63 @@ This installation guide assumes that the following softwares are already install
 

```

Comment:
> No. I wanted "Dependecies" section to be added just before #installation instructions for dependencies.
—@sandoche2k

I believe the section for installation instruction for dependencies is already titled "Dependencies".